### PR TITLE
'solution'

### PR DESCRIPTION
**Pull request by IvanGuculyak**

### CODE REVIEW

**Review by vsmutok — CHANGES_REQUESTED** [review]
fix tests

File: taxi/__pycache__/__init__.cpython-311.pyc
Comment (Grayder0152) [inline taxi/__pycache__/__init__.cpython-311.pyc:1]: Add `__pycache__` directory to gitignore

**Review by Grayder0152 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+     __pycache__
+     venv
+     .pyc
+     .idea
+     sqlite3
+     .md
+     cache
```
Comment (ArsenPidhoretskyi) [inline .gitignore:7]: Add empty line
Also fix tabulation? Why all items have spaces/tabs before?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+     __pycache__
+     venv
+     .pyc
```
Comment (ArsenPidhoretskyi) [inline .gitignore:3]: Wrong name .pyc is mean folder
*.pyc mean files with extension pyc

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Remove pycache from github. Read about git rm --cached

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
pycache still exist inside github
image

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.pytest_cache/
+**__pycache__/
+*.pyc
+**db.sqlite3
```
Comment (vasylhnatiuk) [inline .gitignore:10]: add newline

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**